python/packages/autogen-agentchat/src/autogen_agentchat/agents/_base_chat_agent.py

```diff
@@ -55,7 +55,7 @@ async def on_messages_stream(
     async def run(
         self,
         *,
-        task: str | ChatMessage | None = None,
+        task: str | ChatMessage | List[ChatMessage] | None = None,
```

Comment:
We need to add unit test for assistant agent to validate the behavior of run and run_stream when input is a list of chat messages.